1 table.sql

```diff
@@ -0,0 +1,32 @@
+use master
+go
+drop database if exists BikeDB
+go
+create database BikeDB
+go
+use BikeDB
+go
+drop table if exists BikeShop
+go
+create table dbo.BikeShop(
+BikeId int not null identity primary key,
+CustomerName varchar(50) not null constraint ck_Bike_customername_cannot_be_blank check(CustomerName <> ' '),
+CustomerAddress varchar(100) not null constraint ck_Bike_customeraddress_cannot_be_blank check(CustomerAddress <> ' '),
+PhoneNumber varchar(15) not null constraint ck_Bike_phonenumber_cannot_be_blank check(PhoneNumber <> ' '),
```

Comment:
check that the format of the phone number is 3 numeric, a dash, 3 numeric, a dash 4 numeric

Reply:
No, the constraint should ensure that the values are all numbers separated by dashes ie: [0-9]...[-]